packages/esm-amd-loader/test/package.json

```diff
@@ -7,7 +7,7 @@
     "clean": "rm -rf lib/",
     "lint": "tslint --project . --format stylish",
     "build": "npm run clean && tsc",
-    "test:wct": "npm run lint && npm run build && wct"
+    "test:wct": "npm run lint && npm run build && bash run-wct.sh"
```

Comment:
How does test:wct get invoked?  Shouldn't this be test:integration?

Reply:
esm-amd-loader's `test:integration` command cds to this package and calls this script. i thought that made more sense, since it's the integration test for that package, which just happens to be implemented in a separate package.

Reply:
Got it.  Thanks!